### DISCUSSION TITLE
Plugin UI Disappears in Adobe Premiere

### DISCUSSION THREAD

**Discussion by XTREEMMAK** [apohl79/audiogridder#741] [Help]
Hi there,

Thought I'd give this software a try since the concept is great, however I cant seem to get it to work reliably in Adobe Premiere.

On first load, the plugins and everything will show, however once you close the plugin Window and reopen it, the AG plugin interface as well as the RPD view is completely gone. The only way to get the UI back is to remove the plugin and add it back in again. 

This happens on 1.2.0 (BETA 5). I tried the last stable release and I guess that release was pre RDP implementation. However, I was able to at least get the generic UI to work. On the beta, even the generic UI disappears.

Hope there's a fix for this soon :)

**Comment by apohl79** [comment]
Would you be able to collect some more info for this (https://audiogridder.com/bug-reports/)? I can't test on premiere, as I don't own a license for this software. So no idea whats going on there. Maybe a short screen capturing video would be clearing things up as well.
